src/overlays/actors/ovl_En_Wf/z_en_wf.c

```diff
@@ -137,81 +143,1349 @@ static ColliderCylinderInit D_80B37A5C = {
+    this->action = WOLFOS_ACTION_WAIT;
+    this->actionTimer = (Rand_ZeroOne() * 10.0f) + 2.0f;
+    this->actor.speedXZ = 0.0f;
+    this->actor.world.rot.y = this->actor.shape.rot.y;
+    EnWf_SetupAction(this, EnWf_Wait);
+}
+
+void EnWf_Wait(EnWf* this, GlobalContext* globalCtx) {
+    Player* player;
+    s32 pad;
+    s16 angle;
+
+    player = GET_PLAYER(globalCtx);
+    SkelAnime_Update(&this->skelAnime);
+
+    if (this->headTilt != 0) {
+        angle = (this->actor.yawTowardsPlayer - this->actor.shape.rot.y) - this->headRot.y;
+
+        if (ABS(angle) > 0x2000) {
+            this->headTilt--;
+            return;
+        }
+
+        this->headTilt = 0;
+    }
+
+    angle = this->actor.yawTowardsPlayer - this->actor.shape.rot.y;
+    angle = ABS(angle);
+
+    if (!(EnWf_DodgeRanged(globalCtx, this))) {
+        // Only use of unk_2E0: never not zero, so this if block never runs
+        if (this->unk_2E0 != 0) {
+            this->unk_2E0--;
+
+            if (angle >= 0x1FFE) {
+                return;
+            }
+            this->unk_2E0 = 0;
+        } else if (EnWf_ChangeAction(globalCtx, this, 0)) {
+            return;
+        }
+
+        angle = player->actor.shape.rot.y - this->actor.shape.rot.y;
+        angle = ABS(angle);
+
+        if ((this->actor.xzDistToPlayer < 80.0f) && (player->swordState != 0) && (angle >= 0x1F40)) {
+            this->actor.shape.rot.y = this->actor.world.rot.y = this->actor.yawTowardsPlayer;
+            EnWf_SetupRunAroundPlayer(this);
+        } else {
+
+            this->actionTimer--;
+
+            if (this->actionTimer == 0) {
+                if (Actor_IsFacingPlayer(&this->actor, 0x1555)) {
+                    if (Rand_ZeroOne() > 0.3f) {
+                        EnWf_SetupRunAtPlayer(this, globalCtx);
+                    } else {
+                        EnWf_SetupRunAroundPlayer(this);
+                    }
+                } else {
+                    EnWf_SetupSearchForPlayer(this);
+                }
+                if ((globalCtx->gameplayFrames & 0x5F) == 0) {
+                    Audio_PlayActorSound2(&this->actor, NA_SE_EN_WOLFOS_CRY);
+                }
+            }
+        }
+    }
+}
+
+void EnWf_SetupRunAtPlayer(EnWf* this, GlobalContext* globalCtx) {
+    f32 lastFrame = Animation_GetLastFrame(&gWolfosRunningAnim);
+
+    Animation_Change(&this->skelAnime, &gWolfosRunningAnim, 1.0f, 0.0f, lastFrame, ANIMMODE_LOOP_INTERP, -4.0f);
+    this->action = WOLFOS_ACTION_RUN_AT_PLAYER;
+    EnWf_SetupAction(this, EnWf_RunAtPlayer);
+}
+
+void EnWf_RunAtPlayer(EnWf* this, GlobalContext* globalCtx) {
+    s32 sp5C;
+    s32 sp58;
+    s32 pad;
+    f32 sp50 = 0.0f;
+    s32 pad2;
+    Player* player = GET_PLAYER(globalCtx);
+    s32 pad3;
+    s16 temp_v1;
+    s16 temp_v0;
+    f32 sp3C;
+
+    if (!EnWf_DodgeRanged(globalCtx, this)) {
+
+        Math_SmoothStepToS(&this->actor.shape.rot.y, this->actor.yawTowardsPlayer, 1, 0x2EE, 0);
```

Comment:
same here about the break, personally dont think it helps readability when its right after the opening of an if statement. at least for me it makes it harder to see that a new block has began

theres another one below

Reply:
I think I got all of them...